Source/GSXib5KeyedUnarchiver.m

```diff
@@ -0,0 +1,3391 @@
+                                    //@"NSHeaderClipView",
+                                    @"NSHScroller", @"NSVScroller", @"NSsFlags", @"NSsFlags2",
+                                    @"NSColumnAutoresizingStyle", @"NSTvFlags", @"NScvFlags",
+                                    @"NSSupport", @"NSName",
+                                    @"NSMenuItem",
+                                    @"NSDocView",
+                                    @"NSSliderType",
+                                    @"NSCellPrototype", @"NSBrFlags", @"NSNumberOfVisibleColumns",
+                                    @"NSWhite", @"NSRGB", @"NSCYMK",
+                                    //@"NSContents", @"NSAlternateContents",
+                                    @"NSCellFlags", @"NSCellFlags2",
+                                    @"NSButtonFlags", @"NSButtonFlags2",
+                                    @"NSSelectedIndex", @"NSUsesItemFromMenu",
+                                    @"NSNormalImage", @"NSAlternateImage",
+                                    @"NSBorderType", @"NSBoxType", @"NSTitlePosition",
+                                    @"NSTitleCell", @"NSOffsets",
+                                    @"NSMatrixFlags", @"NSNumCols", @"NSNumRows",
+                                    @"NSSharedData", @"NSFlags", @"NSTVFlags", @"NSDefaultParagraphStyle",
+                                    @"NSpiFlags", nil];
+          RETAIN(XmlKeysDefined);
+
+          // These define XML tags (i.e. '<autoresizingMask ...') to an associated decode method...
+          XmlTagToDecoderSelectorMap = [NSDictionary dictionaryWithObjectsAndKeys:
+                                                       @"decodeTableColumnResizingMaskForElement:", @"tableColumnResizingMask",
+                                                       //@"decodeAutoresizingMaskForElement:", @"autoresizingMask",
+                                                       @"decodeWindowStyleMaskForElement:", @"windowStyleMask",
+                                                       //@"decodeWindowPositionMaskForElement:", @"windowPositionMask",
+                                                       //@"decodeModifierMaskForElement:", @"modifierMask",
+                                                       @"decodeTableViewGridLinesForElement:", @"tableViewGridLines",
+                                                       nil];
+          RETAIN(XmlTagToDecoderSelectorMap);
+
+          // These define XML attribute keys (i.e. '<object key="name" key="name" ...') to an associated decode method...
+          // The associated decode method may process MULTIPLE keyed attributes as in such cases as
+          // decoding the integer flag masks...
+          XmlKeyToDecoderSelectorMap = [NSDictionary dictionaryWithObjectsAndKeys:
+                                                       @"decodeIntercellSpacingHeightForElement:", @"NSIntercellSpacingHeight",
```

Comment:
Most of this file seems to break 80ch/line pretty nastily and confuses Github web UI a lot. (But if we are not sticking to 80ch/line, please ignore this comment).

Does it make sense to have value line + key line + blank line, instead of value+key line?

Reply:
Done